Appraisals

```diff
@@ -1,5 +1,5 @@
 appraise "sass-3-4" do
-  gem "sass", "~> 3.4"
+  gem "sass", "~> 3.4.22"
```

Comment:
I've switched the Bourbon requirement to be `5.0.0.beta.6` or greater and this seems to solve this.